client/blocks/comment-detail/style.scss

```diff
@@ -0,0 +1,261 @@
+%comment-detail__ellipsis {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+}
+
+%comment-detail__gray-link {
+	color: $gray;
+	cursor: pointer;
+	fill: $gray;
+	&:focus,
+	&:hover {
+		color: $blue-wordpress;
+		fill: $blue-wordpress;
+	}
+}
+
+.comment-detail.card {
+	font-size: 14px;
+	margin: 0 auto;
+	padding: 0;
+	transition: margin .15s linear;
+
+	&.is-expanded {
+		margin: 16px auto;
+	}
+	&:not( .is-expanded ):hover {
+		box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20% );
+		z-index: 1;
+	}
+
+	.gridicon {
+		margin-right: 4px;
+		vertical-align: middle;
+	}
+}
+
+.comment-detail__author-avatar {
+	flex-shrink: 0;
+	img {
+		border-radius: 50%;
+	}
+}
+.comment-detail__author-info {
+	@extend %comment-detail__ellipsis;
```

Comment:
Mmh, I'll probably drop both altogether actually.

`%comment-detail__ellipsis` will be an ugly repetition, but eh, I'm kinda confident we won't have to use it many more times.

`%comment-detail__gray-link` this actually is a sort of placeholder.
Like, in this PR the links are all gray and blue on hover (like in the Reader's comments).
But, for example, in the notification panel, on hover the icon remains gray and the label becomes yellow.
Also the various actions will most likely have "states" (e.g. approved might turn green, spam red), leading to an inevitable overhaul once the design is final.